src/html_parser/readability.py

```diff
@@ -36,7 +41,8 @@ def parse(self, input: ParserInput) -> ParserOutput:
         """
         if input.document_source_url is None:
             raise ValueError(
-                "HTML processing was supplied an empty source URL for {input.document_id}"
+                "HTML processing was supplied an empty source URL for {"
+                "input.document_id} "
```

Comment:
is this supposed to be an f-string? (i realise you didn't introduce this if it's a bug)